h/views/api.py

```diff
@@ -175,28 +175,16 @@ def links(context, request):
 
 @api_config(route_name='api.search',
             link_name='search',
-            description='Search for annotations')
+            description='Search for annotations, return legacy json')
```

Comment:
I don't think this is necessary. We don't have any concrete plans as of yet to have the client use the jsonld format vs our own, so our own JSON format is not "legacy" by any means.